.spectral.yml

```diff
@@ -0,0 +1,139 @@
+extends: [spectral:oas2]
+rules:
+  operation-operationId: false
+  operation-tags: info
+  operation-2xx-response: error
+  paths-camelCase:
+    description: All YAML/JSON paths MUST follow camelCase
+    severity: warn
+    recommended: true
+    message: "{{property}} is not camelCase: {{error}}"
+    given: $.paths[*]~
+    then:
+      function: pattern
+      functionOptions:
+        # match: "/^(\/{1}(([{]?[a-z])[A-Za-z0-9]*[}]?)*)+$/" # - more generic one, allows /asasd{asdas}sadas pattern but also not closed braces
+        match: "^\/([a-z][a-zA-Z0-9]+)?(\/[a-z][a-zA-Z0-9]+|\/{[a-z][a-zA-Z0-9]+})*$" # doesn't allow /asasd{asdas}sadas pattern or not closed braces

```

Comment:
We have a [casing function](https://github.com/stoplightio/spectral/blob/develop/docs/reference/functions.md#casing) in 5.0-beta (develop branch right now) if you fancy giving it a try.

Reply:
Cool! Planning `Hyphenated-Pascal-Case` for Headers?

Reply:
also, I guess paths won't match casing functions well, will they?